packages/replayio/src/commands/upload-source-maps.ts

```diff
@@ -7,7 +7,6 @@ import { dim } from "../utils/theme";
 import { logPromise } from "../utils/async/logPromise";
 
 registerCommand("upload-source-maps <paths...>", { requireAuthentication: true })
-  .alias("upload-sourcemaps")
```

Comment:
Subjective change, I guess, but this...
![Screenshot 2024-04-12 at 10 05 55 AM](https://github.com/replayio/replay-cli/assets/29597/91b13b99-101d-4e94-ade8-04f96d7aae79)

...is a better first impression than this...
![Screenshot 2024-04-12 at 10 04 16 AM](https://github.com/replayio/replay-cli/assets/29597/50e8f85d-21f7-49f9-9ea3-da53b627b5b4)

This isn't a command people will be running often, so I think we can just ask them to type it correctly :)

Reply:
The only alternative that comes to mind is to drop the aliases from this table altogether (see my comment [here](https://github.com/replayio/replay-cli/pull/375#discussion_r1562613521)) - that is if we want to keep it being available to people.

Reply:
I think we should just do a clean break. We've already done this with so many things (`ls` -> `list`, `rm` -> `remove` and `rm-all` -> `remove --all`, etc.) I don't see a reason to carry the old alias forward :)

Reply:
My thinking here was a mix of keeping the old name and that people (us included :P) use that spelling interchangeably - it's actually the latter that made me add the alias. I'm fine with removing it though 👍